### PR TITLE
Json - New content: remove object from json array, validate json

### PR DESCRIPTION
**Pull request by tethryus**
Added test cases.

### CODE REVIEW

**Comment by mihaitusa** [comment]
@tethryus  This commit should contain files related to JSON content and not related to Circle CI

File: content/io/cloudslang/base/json/remove_object_from_json_array.sl [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#   (c) Copyright 2015 Hewlett-Packard Development Company, L.P.
```
Comment (mihaitusa) [inline content/io/cloudslang/base/json/remove_object_from_json_array.sl]: replace with ... (c) Copyright 2016 Hewlett-Packard Enterprise Development Company, L.P. ... in all doc's

File: content/io/cloudslang/base/json/remove_object_from_json_array.sl [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - json_object
+    - index:
+        required: false
+  action:
+    python_script: |
+      try:
+        import json
+        decoded_json_array = json.loads(json_array)
+        decoded_json_object = json.loads (json_object)
+        index = locals().get('index')
```
Comment (mihaitusa) [inline content/io/cloudslang/base/json/remove_object_from_json_array.sl]: Get rid of locals().get('whatever')... is not needed

File: content/io/cloudslang/base/json/remove_object_from_json_array.sl [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+####################################################
+
+namespace: io.cloudslang.base.json
+
+operation:
+  name: remove_object_from_json_array
+  inputs:
+    - json_array
+    - json_object
+    - index:
```
Comment (mihaitusa) [inline content/io/cloudslang/base/json/remove_object_from_json_array.sl]: add here default: None

File: content/io/cloudslang/base/json/validate_json.sl [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+####################################################
+#!!
+#! @description: Validate json input.
+#! @input json_input: JSON from which to retrieve keys - Example: '{"k1": {"k2": {"k3":"v3"}}}'
```
Comment (mihaitusa) [inline content/io/cloudslang/base/json/validate_json.sl]: I don't think the description is appropriate here.

File: test/io/cloudslang/base/json/test_remove_object_from_json_array.sl [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            - json_array
+            - json_object
+            - index
+
+        publish:
+          - json_output
+
+        navigate:
+          - SUCCESS: test_equality
+          - FAILURE: CREATEFAILURE
```
Comment (mihaitusa) [inline test/io/cloudslang/base/json/test_remove_object_from_json_array.sl]: replace with CREATE_FAILURE to be more snake_case....

**Comment by mihaitusa** [comment]
@tethryus  Please handle the comments so we can merge it!

File: content/io/cloudslang/base/json/remove_object_from_json_array.sl (hunk 3 above)
Comment (tethryus) [inline content/io/cloudslang/base/json/remove_object_from_json_array.sl]: default: None added. thanks for the input.

**Comment by tethryus** [comment]
Thanks for pointing the issues out. Addressed all the comment. Thanks for the input on all issues discoevered. Everything should be ok and ready to merge now.

**Pull request merged**